index.js

```diff
@@ -1,6 +1,6 @@
 require('babel-core/register')
 require('ts-node').register()
-require('coffee-script/register')
+require('coffeescript/register')
```

Comment:
This is due to bumping artsy passport, which finally removed this outdated version. This does mean that this PR may have updated the version of `coffeescript` that Force is using. Such is life.